src/views/ProposalList.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <b-col>
```

Comment:
This template is largely what I pulled over from the backend project, just refactored to get data differently and updated the form.